ApplicationPattern.yaml

```diff
@@ -465,27 +465,27 @@ paths:
                       type: string
```

Comment:
Can we update property descriptions of /v1/bequeath-your-data-and-die
1. new-application-name : can we update the description to "find [http-server-interface-capability/application-name]), because the request-body shall expected to contain application-name == http-server/application-name.
2. other attribute's description to "update". 
Kindly correct me if I am wrong

Reply:
In the _/v1/bequeath-your-data-and-die_ there is a concrete UUID.
It always points directly towards the new release.
There is always exactly one instance.
I would propose to change for "update" in all cases.

Your opinion?

Reply:
I agree with your point for change for "update" to new-application-release, new-application-protocol, new-application-address and new-application-port. I have the following thought for **line 41 : description for property _new-application-name_** . 
1. Currently, it is pointing to http-c/application-name = NewRelease, but the actual request-body is intended to contain "actual application-name of Target Application". So, the current description shall be changed.
2. Initially, I thought, the value of _NewApplicationName_  should be equal to http-server/application-name, but now I remember, since application-name of an application can change in future, the description for "new-application-name" cannot be pointing to "http-server/application-name". 

My proposal:
Since, the release-number and other attributes are correctly pointing to *-1-0-0-tcp-c-nr-1-0-0-000, can we remove the current description in line 41 for new-application-name (second line of description) , since it shall mislead in future that the "new-application-name" attribute value will be "NewRelease" (as per http-client/application-name of *-1-0-0-tcp-c-nr-1-0-0-000).

Kindly correct my proposal if I am wrong.

Reply:
I am not 100% sure what you mean, but the /v1/bequeath-your-data-and-die has attributes for informing the OldRelease about where to handover the data. _new-application-name_ is one of the attributes. The value of this attribute has to be put somewhere.